dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -6798,7 +6798,7 @@ private void validateFileAsset(final Contentlet contentlet, final String content
      * @param contentlet
      */
     private void validateSite(Contentlet contentlet) {
-        if (Config.getBooleanProperty("site.key.dns.validation", false)) {
+        if (Config.getBooleanProperty("site.key.dns.validation", true)) {
```

Comment:
this change is breaking a few tests

Reply:
@erickgonzalez I just reverted this change.  I'll re-introduce it on a separate PR  (for the sake of keeping this one small)